src/js/popupDom.js

```diff
@@ -0,0 +1,83 @@
+/** @format */
+
+const popupDom = (item) => {
+  const popupSection = document.getElementById('popup');
+
+  const popupContainer = document.createElement('div');
+  popupContainer.id = Number(item.id) + 1;
+  popupContainer.classList.add('meme-popup', 'hidden');
+
+  const closeBtn = document.createElement('button');
+  closeBtn.classList.add('close-btn', 'fas', 'fa-times');
+
+  const image = document.createElement('img');
+  image.src = item.url;
+  const imageTitle = document.createElement('p');
+  imageTitle.classList.add('image-title');
+  imageTitle.innerHTML = item.name;
+
+  // ! Comment Container
+  const commentContainer = document.createElement('div');
+
+  const commentTitle = document.createElement('h4');
+  commentTitle.innerHTML = 'Comments<span class="comments-number"></span>';
+
+  const commentTexts = document.createElement('div');
+  commentTexts.classList.add('comment-container');
+  commentContainer.append(commentTitle, commentTexts);
+
+  // 61581, 176908,   //error 157978092, 89655, 7183956, 71428573
+  //! API SECTİON
+  const URL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/GuhYPExBlAlZ5RuYo3CN/comments?item_id=';
+  const getMemeData = async () => {
+    const response = await fetch(`${URL}${Number(item.id)}`);
+    const data = await response.json();
+    if (response.status === 200) {
+      data.forEach((item) => {
+        const p = document.createElement('p');
+        p.classList.add('comment-text');
+        p.innerHTML = `${item.creation_date} ${item.username}: ${item.comment}`;
+        commentTexts.append(p);
+      });
+    } else {
+      commentTexts.innerHTML += '<p>No comments</p>';
+    }
+  };
+
+  getMemeData();
+
+  // ! Form Container
+  const formContainer = document.createElement('div');
```

Comment:
- [ ] Optional
Right now an empty comment is also submitted. We can add the validation for the form to prevent empty comments.